docs-build/deploy.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+set -e
+mkdir ./tmpdocclone
+cd ./tmpdocclone
+git clone https://github.com/${GH_ORG}/docs.git
+
+cd docs
+git checkout -b test${TRAVIS_BUILD_NUMBER}
+
+cp ../../build/cli-commands.md ./docs/using-appsody/cli-commands.md
+
+git add docs/using-appsody/cli-commands.md
+
+git commit -m "Travis build: $TRAVIS_BUILD_NUMBER"
```

Comment:
To avoid problems with the CLA assistant agreement, add this argument to the commit command:
`--author="Kyle G. Christianson <christik@us.ibm.com>"`
Please do this for the other deploy script as well.

Reply:
I tried this locally.  It seemed to work. We should review with the team.